google/cloud/bigtable/examples/bigtable_samples_instance_admin_ext.cc

```diff
@@ -0,0 +1,305 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "google/cloud/bigtable/instance_admin.h"
+#include "google/cloud/bigtable/instance_admin_client.h"
+#include <google/protobuf/text_format.h>
+#include <typeindex>
+#include <typeinfo>
+
+namespace btproto = ::google::bigtable::admin::v2;
```

Comment:
I do not think we use this alias, if so, please remove.

Reply:
fixed.